src/data/valid/Database-nom_calibration_example.yaml

```diff
@@ -0,0 +1,56 @@
+data_generation_set:
+- id: nmdc:omprc-11-0003fm52
```

Comment:
For example data files, what are the advantages and disadvantages of using omprc as the typecode on an `MassSpectrometry` instance? I would think that we would want to use the `MassSpectrometry`-specific typecode.

Reply:
This is the result of copying and then modifying an existing record from mongo prod. I can update the typecode here.